server/main.lua

```diff
@@ -283,45 +289,57 @@ QBCore.Functions.CreateCallback('qb-phone:server:GetPhoneData', function(source,
 end)
 
 QBCore.Functions.CreateCallback('qb-phone:server:PayInvoice', function(source, cb, society, amount, invoiceId, sendercitizenid)
-    local Invoices = {}
     local Ply = QBCore.Functions.GetPlayer(source)
     local SenderPly = QBCore.Functions.GetPlayerByCitizenId(sendercitizenid)
-    local invoiceMailData = {}
-    if SenderPly and Config.BillingCommissions[society] then
-        local commission = round(amount * Config.BillingCommissions[society])
-        SenderPly.Functions.AddMoney('bank', commission)
-        invoiceMailData = {
-            sender = 'Billing Department',
-            subject = 'Commission Received',
-            message = string.format('You received a commission check of $%s when %s %s paid a bill of $%s.', commission, Ply.PlayerData.charinfo.firstname, Ply.PlayerData.charinfo.lastname, amount)
-        }
-    elseif not SenderPly and Config.BillingCommissions[society] then
-        invoiceMailData = {
-            sender = 'Billing Department',
-            subject = 'Bill Paid',
-            message = string.format('%s %s paid a bill of $%s', Ply.PlayerData.charinfo.firstname, Ply.PlayerData.charinfo.lastname, amount)
-        }
-    end
-    Ply.Functions.RemoveMoney('bank', amount, 'paid-invoice')
-    exports['qb-phone']:sendNewMailToOffline(sendercitizenid, invoiceMailData)
-    exports['qb-banking']:AddMoney(society, amount, 'Phone invoice')
-    MySQL.query('DELETE FROM phone_invoices WHERE id = ?', { invoiceId })
-    local invoices = MySQL.query.await('SELECT * FROM phone_invoices WHERE citizenid = ?', { Ply.PlayerData.citizenid })
-    if invoices[1] ~= nil then
-        Invoices = invoices
+    local invoiceMailData = nil
+    if Ply then
+        local exists = MySQL.query.await('select count(1) FROM phone_invoices WHERE id = ? and citizenid = ?', { invoiceId, Ply.PlayerData.citizenid })
```

Comment:
Maybe we could use `AS count` just to make the column read nicer when indexing, like count of results.
It'd turn it from `exists[1]["count(1)"]` to `exists[1]["count"]`.

I thought about `AS result`, but that could become confusing/inconsistent with our other usage of queries. It would definitely make sense alone though.

Reply:
I mean, yeah. Wont really make any difference really, but I do see what you mean.

Reply:
Of course, just readability.